v2/cf-nodejs-logging-support/src/lib/logger/record-factory.ts

```diff
@@ -48,13 +49,13 @@ export default class RecordFactory {
 
     // init a new record and assign fields with output "req-log"
     static buildReqRecord(req: any, res: any): any {
-
+        const level = (req.logger as Logger).getLoggingLevel();
         const requestAccessor = RequestAccessor.getInstance();
         const responseAccessor = ResponseAccessor.getInstance();
 
         const configInstance = Config.getInstance();
         const reqLogFields = configInstance.getReqFields();
-        let record: any = { "level": "info" };
+        let record: any = { "level": level };
```

Comment:
The level we get from 
```js
(req.logger as Logger).getLoggingLevel();
``` 
is a threshold value and not the actual logging level of the record, right? Request logs are typically logged at "info" level. If the threshold is set to "error", we do not expect any request logs to get logged. I think it would be great if we could set the default logging level for request logs via config.